test/penalizer/Penalizer.test.ts

```diff
@@ -290,6 +273,81 @@ contract('Penalizer', function ([relayOwner, relayWorker, relayManager, otherAcc
     })
   })
 
+  describe('should receive/reject claims according with the commitment time', function () {
+    before(async function () {
+      await penalizer.setHub(relayHub.address, { from: await penalizer.owner() })
+    })
+    beforeEach(async () => {
+      await token.mint('1000', forwarder.address)
+
+      await relayHub.stakeForAddress(relayManager, 1000, {
+        from: relayOwner,
+        value: ether('1'),
+        gasPrice: gasPrice
+      })
```

Comment:
you shouldn't need to stake eth before each of these tests, no? but rather, only when a manager is penalized and its stake is slashed. 

if possible let's have this code as a before/after statement for the successful claims only. this will save time, gas, and reduce the amount of things that can go wrong

Reply:
Agreed, thanks for the suggestions. It's not required when the claim call is rejected.